### PR TITLE
Add the notion of a fatal vs non-fatal global outage (+ tests for the…

### LINKED ISSUE #64: For Global Status Indicator, revise outage categories
If the website is down **or** both SOLR and Articles are down, that's a "fatal" error category (red circle + alternative "fatal" text).

If **either** SOLR or Articles is down (but not both), then give the "critical" status (red circle, with "critical" text that is distinct from "fatal").

Any other single outage should give a yellow "issues" status.

Currently the text is the same for all "critical" statuses, and the global status text is the same as that in the table. We have the ability to display a special `global_message` for the top-level status indicator through the config, but also need to check for the new, proposed, "fatal" category and have a specific message for that.

### PR DESCRIPTION
**Pull request by jkeck**
… GlobalStatusSummary component)

Closes #64 

### Website (same as SOLR & EDS)
website-fatal

---

### SOLR & EDS (same as website)
non-website-fatal

## Non-Fatal Outage
outage

### Critical Service (same as non-critical)
critical-issue

---

### Non-Critical Service (same as critical)
non-critical-issue

### CODE REVIEW

File: src/components/GlobalStatusSummary.jsx [hunk 1]
```diff
@@ -8,15 +8,29 @@ const statusReducer = (endpointStatuses) => {
...
+  const anyFatalOutages = (() => {
+    const appStatus = endpointStatuses.searchworksApplication.status;
+    const solrStatus = endpointStatuses.swSolr.status;
+    const articleStatus = endpointStatuses.ebsco.status;
+
+    if (appStatus === 'outage' || (solrStatus === 'outage' && articleStatus === 'outage')) {
+      return true;
+    }
+    return false;
+  })();
```
Comment (jkeck) [inline src/components/GlobalStatusSummary.jsx]: Not sure if this anonymous function assigned to a constant and invoked inline is the best way to accomplish this sort of thing.

Comment (mejackreed) [reply src/components/GlobalStatusSummary.jsx]: Yeah.. perhaps we could abstract class to a POJO? I think what you have could work for now, but that could separate the globabl status logic from the view?

Comment (jkeck) [reply src/components/GlobalStatusSummary.jsx]: I ❤️ POJOs.  BRB, gotta do some refactorin'

**Comment by camillevilla** [comment]
This PR also takes care of some of the tests listed in #68

**Pull request merged**